app-misc/vcontrold/files/man_generation.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/doc/man/CMakeLists.txt b/doc/man/CMakeLists.txt
+index 65024b3..d503ff1 100644
+--- a/doc/man/CMakeLists.txt
++++ b/doc/man/CMakeLists.txt
+@@ -14,14 +14,14 @@ if(VSIM)
+ endif(VSIM)
+ 
+ foreach(MANUAL IN LISTS MANUALS)
+-    set(MANPAGE_OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${MANUAL}.1.gz)
++    set(MANPAGE_OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${MANUAL}.1)
+     set(MANPAGE_SOURCE ${CMAKE_CURRENT_SOURCE_DIR}/${MANUAL}.rst)
+     add_custom_command(OUTPUT ${MANPAGE_OUTPUT}
+-        COMMAND ${RST2MAN} ${RST2MAN_OPTS} ${MANPAGE_SOURCE} | gzip > ${MANPAGE_OUTPUT}
++        COMMAND ${RST2MAN} ${RST2MAN_OPTS} ${MANPAGE_SOURCE} ${MANPAGE_OUTPUT}
+         DEPENDS ${MANPAGE_SOURCE}
+         BYPRODUCTS ${MANPAGE_OUTPUT}
```

Comment:
I think this is what makes ninja unhappy. How can the same file be output and byproduct at the same time?